querydsl-codegen/src/main/java/com/mysema/query/codegen/DefaultVariableNameFunction.java

```diff
@@ -15,14 +15,15 @@
  */
 package com.mysema.query.codegen;
 
+import com.google.common.base.Function;
 import com.mysema.codegen.StringUtils;
 import com.mysema.util.JavaSyntaxUtils;
 
 /**
  * Default variable name generation strategy which un-capitalizes the first letter of the class name.
  *
  */
-public final class DefaultVariableNameFunction implements NamingFunction {
+public final class DefaultVariableNameFunction implements Function<EntityType, String> {
```

Comment:
the backwards compatibility checker didn't like it, although I think it's compatible